### ISSUE TITLE
feat: Per event selectable template

### ISSUE THREAD

**Issue by jaspnas** [Datateknologerna-vid-Abo-Akademi/date-website#463]
**Is your feature request related to a problem? Please describe.**

Currently the only custom event template (on DaTe) is the hard-coded override for årsfest

**Describe the solution you'd like**

Allow per-event selectable template defaulting to the standard template

**Additional context**

Can be accomplished by reading the template names from filesystem and creating a drop-down in admin to select between them.

**Comment by andreedd** [comment]
Yep good issue, I though about this at some point, and would make it much more dynamic and less hardcoded.

**Comment by theoabw** [comment]
Will try to prioritize this this year

**Comment by theoabw** [comment]
Done: `Event.template` CharField with choices (events/models.py:60), selectable in admin, with fallback to the legacy title/slug mappings. Implemented in #938.
